inception-project/inception
# #1781 Update Monitoring/Agreement section

Updated monitoring section, agreement is ok.

**How to test manually**
* Build doc
* Read monitoring section to make sure that it is current to inception, not webanno

**Automatic testing**

**Documentation**
